packages/monorepo-split-github-action/action.yaml

```diff
@@ -10,21 +10,24 @@ inputs:
         description: 'Remote organization'
         required: true
     split-repository-name:
-        description: 'Remove repository'
+        description: 'Remote repository'
         required: true
     commit-message:
-        description: '[Optional] commit message for the output repository. ORIGIN_COMMIT is replaced by the URL@commit in the origin repo'
-        default: 'Package split ORIGIN_COMMIT'
+        description: '[Optional] commit message for the output repository. ORIGIN_COMMIT is replaced by the URL@commit in the origin repo.'
         required: false
+    branch:
+        description: 'Git branch'
+        default: 'main'
+        required: true
```

Comment:
Do you know any simple way to switch all `master` branches to `main`?

I'd have to update all the projects now.
Maybe go with `master` now and handle rename in new PR

Reply:
Switched to master.